docs/UserGuide.md

```diff
@@ -93,80 +91,49 @@ Examples:
-* The index **must be a positive integer** 1, 2, 3, …​
-
-Examples:
-* `list` followed by `done 2` marks the 2nd task in the displayed task list as done.
-* `find essay` followed by `done 1` marks the 1st task in the results of the `find` command as done.
+Format: `list`
 
-### Editing a task: `edit`
+##### Editing a task: `edit`
 
 Edits an existing task’s details in TR4CKER.
 
 Format: `edit INDEX [n/NAME] [dl/DEADLINE] [des/TASKDESCRIPTION]`
 
-* Edits the task's name, deadline and description at the specified `INDEX`.
+* Edits the task's name, deadline and description at the specified `INDEX`. 
 * The index refers to the index number shown in the task list.
 * The index **must be a positive integer** 1, 2, 3, …​
 * At least one of the optional fields must be provided.
 * Existing task’s details will be updated to the new task’s details being inputted.
-* Editing of tags can be done using `tag` command.
 
 Examples:
-* `edit 1 n/prepare for tP tasks` - Edits the description of the 1st task to be `prepare for tP tasks`.
-* `edit 2 dl/2020-09-13 1930` - Edits the deadline time of the 2nd task to be 13 Sep 2020, 1930 hrs.
+* `edit 1 n/ prepare for tP tasks` - Edits the description of the 1st task to be `prepare for tP tasks`.
+* `edit 2 dl/ 2020-09-13 1930` - Edits the deadline time of the 2nd task to be 13 Sep 2020, 1930 hrs.
 
-### Editing tags: `tag`
+##### Locating tasks by name: `find`
 
-Adds and/or deletes tag(s) from an existing task in TR4CKER.
+Finds persons whose names contain any of the given keywords.
 
-Format: `tag INDEX [new/NEW_TAG]…​ [del/TAG_TO_DELETE]…​`
+Format: `find KEYWORD [MORE_KEYWORDS]`
 
-* Adds tag(s) to task at the specified `INDEX` when using `new/NEW_TAG`.
-* Deletes tag(s) from task at the specified `INDEX` when using `del/TAG_TO_DELETE`.
-* The index refers to the index number shown in the task list.
-* The index **must be a positive integer** 1, 2, 3, …​
-* At least one of the fields must be provided.
-* Tag(s) will only be added if it does not exist yet.
-* Tag(s) will only be deleted if it exists.
-* Adding and deleting of tags can be used concurrently.
-* Multiple tags can be added and deleted at the same time.
-
-Examples:
-* `tag 1 new/urgent` - Adds a new tag `urgent` to the existing tags of 1st task (if the tag does not already exist).
-* `tag 2 del/assignment` - Deletes the tag `assignment` from the 2nd task (if the tag exists).
-* `tag 3 new/urgent del/assignment new/graded` - Adds 2 new tags `urgent` and `graded`, deletes the tag `assignment`
-from the 3rd task.
-
-### Locating tasks by keyword: `find`
-
-Finds tasks whose names contain any of the given keywords.
-
-Format: `find [KEYWORD_1] [KEYWORD_2] ...`
-
-* The search is case-insensitive. e.g `cs2101` will match `CS2101`
-* The order of the keywords does not matter. e.g. `Presentation CS2101` will match `CS2101 Presentation`
+* The search is case-insensitive. e.g `hans` will match `Hans`
```

Comment:
should this section keep the original to reflect tr4cker instead of addressbook?